### DISCUSSION TITLE
New UI design: Not great

### DISCUSSION THREAD

**Discussion by ruilvo** [community/community#58608] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

The new UI design goes in-line with the most common and "modern" guidelines to make UI/UXs.

But, are they useful?

This is how the toolbar looks now:

![imagem](https://github.com/community/community/assets/8402646/a67c6204-9163-4442-97d4-9dcc2d6f558d)

There is over half of empty space. Buttons got larger. In the meanwhile, if i want to access my usual buttons: Home, issues, PRs, discussions, I have now to navigate a menu or go to small, unlabelled icons, instead of a general flick of the mouse into a target that had a huge activation area, and that I could see beforehand.

It's less navigable, slower to use, harder to discover. We gained nothing from this design.  Swapping iconography for an actual icon+text or text doesn't achieve anything in terms of UX. If may look good on designer presentations, it's not good to use.

**Comment by Fuco1** [comment]
People come to GitHub to do work.  Please let me do work:

- Show me more. I want to see as much as possible without scrolling.  Please get rid of all the empty space.  Everyone has *wide*screen displays now.  Vertical space is precious.  I'm reading code here, 10 lines of context is not enough.
- Make common actions simple.  Please.  I don't come to GitHub to waste time or watch ads (there are no ads).  Why do you make me waste time?

**Reply by PikaMug** [reply]
> Please get rid of all the empty space. Everyone has widescreen displays now.

Everyone except mobile users in portrait. Granted, that's no excuse not to make the desktop experience better, but I imagine it's probably the reason.

**Reply by rybak** [reply]
> Everyone has widescreen displays now.

The navigation being moved all the way to the left feels out of place. Repositories:

![screenshot of a repository; highlighting how Code/Pull Requests/Issues/etc are moved too much to the left](https://github.com/community/community/assets/624072/a0ec266a-a838-494b-9cb1-b16e2b6d31ef)

And profiles:

![screenshot of a user profile; highlighting how Overview/Repositories/Projects/Packages/Stars are moved too much to the left](https://github.com/community/community/assets/624072/629e5020-85e4-4538-ab5d-b941e49fcb77)

Both screenshots are from a 2560×1440 monitor, one step above the 1920×1080 in the common 16:9 resolutions. People with 18:9 and wider monitors probably won't be able to use GitHub in a maximized window.

It would have been nice if the tabs were way closer to the central column.

**Comment by JohnnySn0w** [comment]
Yeah, I just saw this redesign. Some critiques follow:

- Moving anything in a UI should have considerable reasoning behind it. The search bar movement does not seem necessary. People want things to maintain their placement as closely as possible, because our minds internalize those placements for quicker reference in the future. The other people in this thread have already touched on why hiding things that were visible, is a confusing move.

- The box you're holding the forward slash in, isn't properly centered about the slash, it almost looks like a custom glyph on some screens. 
![image](https://github.com/community/community/assets/33181572/35a7c144-9738-466f-8de5-6e921c8df8ef)

- The hamburger menu is awkward looking in its placement , but I'll be honest, I wouldn't know what to do there.

- Finally, there's redundancy in the buttons that were added.

  In the dashboard view, both the GitHub Icon and the new "Dashboard" button go to the same place, but are separate buttons.
  ![image](https://github.com/community/community/assets/33181572/9bbb948c-3796-44e8-aefb-ce02c8bb6a7b)
  
  Likewise, in the community area, the "Overview" tab and the new "Community" button both go to the same location, and are similarly very close to each other. 
  ![image](https://github.com/community/community/assets/33181572/305d22ee-e4a4-432e-b681-698b62f0218e)

I will say, I do think it is cleaner from an aesthetic perspective, but UX should be a priority here. Not that these are massive changes, but I am curious as to the underlying motivation.

**Reply by PikaMug** [reply]
> In the dashboard view, both the GitHub Icon and the new "Dashboard" button go to the same place, but are separate buttons.

I feel this makes sense once users realize the button represents the current page, but is also pretty unnecessary since the h2 header already says what the page is, and in bigger text. I'd much rather it just say "Github - tagline" for those in fullscreen. Speaking of text, it feels a little uncanny how small the button text is next to the logo. I get that it's probably supposed to align with the right-side UI, but still.

**Comment by the-officialjosh** [comment]
You have the option to disable all of these new features. In the feature preview which you can access through the dropdown menu after clicking on your profile picture located at the top right corner of the screen. From there, you can deselect any new features that you don't want to enable.

**Reply by innocentius** [reply]
"You can disable it" is not a solution to "this sucks".

**Reply by the-officialjosh** [reply]
I want to clarify that the phrase 'this sucks' was not mentioned anywhere in the question. Additionally, I would like to emphasize that I am a fellow member of the community who shares the sentiment of not being satisfied with the design. However, I was informed by another individual that these features are currently being tested to gather feedback from users. Regarding your response, it came across as condescending to me, and I wanted to express that I did not appreciate it.

**Reply by innocentius** [reply]
What you might not realize is that this question belongs to the section "feed", which is an ongoing attempt by github to transform the platform into a distracting social media platform. You claimed that "You have the option to disable all of these new features.", while the explore sections, "for you" sections, are not within such options in the "feed." When a question is asked, remember to consider the context when forming your response.

Github has failed to demonstrate the utility provided by such features, they are cumbersome, distracting, heavily unrelated to our activites. What's more, Github continue to fail to address the dissatisfaction of users who repeatedly voiced their concerns in the same feedback section. If these are to "gather feedback from users", then why not github listen to the already gathered response and provide changes accordingly?

Regarding my response, you clearly demonstrated an unsatisfactory understanding of the current situation, and your misinformed response came across as condescending to many users like me. Let me be straight forward: you are not the only one knows that these can be turned off, for now. If my response seemed condescending, then it has fulfilled its purpose.

**Reply by the-officialjosh** [reply]
You do realise I am not a GitHub staff right? Well if being rude make yiu happy well have at it 🫡

**Reply by ltworf** [reply]
I can no longer disable it. It got enabled and that's it.

Wow. A slower ui where stuff is hidden.

Can't we just pay welfare for designers instead of giving them meaningless redesign jobs to do, to justify their position?

**Reply by Fuco1** [reply]
Yes please, I would donate to have this reverted.

**Reply by rjgotten** [reply]
> I can no longer disable it. It got enabled and that's it.

Yeah; it went live and the responsible developers and designers don't seem to have done one iota with the provided feedback during beta on why this thing is bad. (It's also broken in at least one way on Firefox related to nested scroll containers. _Again._)

**Comment by Memorablea** [comment]
But I think this design is very good, thank you!

**Comment by ghost** [comment]
I personally like the new ui

**Comment by johnjago** [comment]
This part of the new UI tends to confuse me. The profile picture is next to the repository name, making me think for a second that it may be the username.

<img width="355" alt="Screenshot 2023-07-08 at 23 10 50" src="https://github.com/community/community/assets/19847480/7c0c0754-010c-426c-be97-eea2668fa0f1">

**Reply by reeveng** [reply]
How can a company like github push these changes without testing them first?

![KEKW](https://cdn.7tv.app/emote/60a487509485e7cf2f5a6fa7/4x.webp)

**Comment by Universalizer** [comment]
Where is the Repository's > "edit description info" option, in these new UI.

**Reply by rybak** [reply]
Seems to be where it always has been for me, the gear icon ⚙️ button on the right side in the "About" section:

![where button is located](https://github.com/community/community/assets/624072/7e596fbb-94f9-4e81-a9eb-a86488b521ea)

**Reply by Universalizer** [reply]
> Seems to be where it always has been for me, the gear icon ⚙️ button on the right side in the "About" section:

When i posted my this previous comment, at that particular time, all of sudden, their were very vast changes in github.com's user interface, after then i had adjust some preferences in my account settings. So I'm back with my previous settings within a few days. Just like seen in the screenshot.

Thank you @rybak for your concern.

**Reply by Universalizer** [reply]
I want to see `Watch/Unwatch, Numbers of Forks And Stargazer buttons` On each and every sub-page of Repository like it was previously before. Every github user wants it's own choice of option, please bring back it again, so give the user options and choice.

**Comment by Siedlerchr** [comment]
The contrast sucks, it' all 50 shades of gray now. Not everyone is using dark theme And the search is now at the right side which is a long way with the mouse. Before it was left/centered

**Comment by OrionCodeDev** [comment]
AAAAAAAAAA stop pushing that shit updates. That new UI design looks terrible. First sort index, now that. You must fire Managers, you lost users...

**Reply by ltworf** [reply]
I'm slowly moving to codeberg.

**Comment by softgripper** [comment]
I don't know how to use the github homepage any more - the only thing it does is remind me to unstar a bunch of repos I no longer care about.

It's horrible.

Recommendations/feeds are good for some sites, this is not one of them.

**Comment by TheDevMinerTV** [comment]
How do I create releases? Before this change it was visible right there in the right side bar and in the top navbar, now it's under `n tags` -> `Releases`.

Unusable.
